src/services/item/entities/Item.ts

```diff
@@ -103,6 +105,7 @@ export class Item<T extends ItemTypeEnumKeys = ItemTypeEnumKeys> extends BaseEnt
     default: ItemType.FOLDER,
     nullable: false,
     enum: Object.values(ItemType),
+    type: 'varchar',
```

Comment:
For this, it matches the DB type but actually we should maybe have it as an enum ? Not to change now, but would have been better typed for the db I think.